src/subdomains/analytics/application/mappers/coin-tracking-history-dto.mapper.ts

```diff
@@ -0,0 +1,76 @@
+import { Asset, AssetCategory } from 'src/shared/models/asset/asset.entity';
+import { Deposit } from 'src/subdomains/staking/domain/entities/deposit.entity';
+import { Reward } from 'src/subdomains/staking/domain/entities/reward.entity';
+import { Withdrawal } from 'src/subdomains/staking/domain/entities/withdrawal.entity';
+import { DepositStatus, RewardStatus, WithdrawalStatus } from 'src/subdomains/staking/domain/enums';
+import { CoinTrackingCsvHistoryDto } from '../dto/output/coin-tracking-history.dto';
+import { CoinTrackingTransactionTypes } from '../services/staking-history.service';
+
+export class CoinTrackingHistoryDtoMapper {
+  static getStakingDepositHistoryCT(deposits: Deposit[]): CoinTrackingCsvHistoryDto[] {
```

Comment:
Use simpler method names. E.g. `mapDeposits` should be sufficient. The rest is part of the class name.